closure/goog/testing/net/xhrio.js

```diff
@@ -467,7 +487,8 @@ goog.testing.net.XhrIo.prototype.send = function(
  * @protected
  */
 goog.testing.net.XhrIo.prototype.createXhr = function() {
-  return goog.net.XmlHttp();
+  return this.xmlHttpFactory_ ? this.xmlHttpFactory_.createInstance() :
+                                goog.net.XmlHttp();
```

Comment:
See [comment below](https://github.com/google/closure-library/pull/797/files#r102348022). By the way this seems to be the only case where `goog.net.XmlHttp` is called in this module.`